docs/runbooks/PersistentVolumeFillingUp.md

```diff
@@ -0,0 +1,158 @@
+<!--
+Original source from KubePersistentVolumeFillingUp
+https://github.com/prometheus-operator/runbooks/blob/main/content/runbooks/kubernetes/KubePersistentVolumeFillingUp.md
+-->
+
+# PersistentVolumeFillingUp
+
+## Meaning
+
+There can be various reasons why a volume is filling up.
+This runbook does not cover application specific reasons, only mitigations
+for volumes that are legitimately filling.
+
+As always refer to recommended scenarios for given service.
+
+## Impact
+
+Service degradation, switching to read only mode.
+
+## Diagnosis
+
+Check app usage in time.
+Check if there are any configurations such as snapshotting, automatic data retention.
+
+## Mitigation
+
+### Data retention
+
+Deleting no longer needed data is the fastest and the cheapest solution.
+
+Ask the service owner if specific old data can be deleted.
+Enable data retention especially for snapshots, if possible.
+
+### Data export
+
+If data is not needed in the service but needs to be processed later
+then send it to somewhere else, for example to S3 bucket.
+
+### Data rebalance in the cluster
+
+Some services automatically rebalance data on the cluster when one node
+fills up.
+Some allow to rebalance data across existing nodes, the other may require
+adding new nodes.
+If this is supported then increase number of replicas and wait for data
+migration or trigger it manually.
+
+Example services that support this:
+
+- cassandra
+- ceph
+- elasticsearch/opensearch
+- gluster
+- hadoop
+- kafka
+- minio
+
+**Notice**: some services may require special scaling conditions such as
+adding twice more nodes than exist now.
+
+### Direct Volume resizing
+
+If volume resizing is available, it's easiest to increase the capacity of
+the volume.
+
+To check if volume expansion is available, run this with your namespace
+and PVC-name replaced.
+
+```shell
+$ kubectl get storageclass `kubectl -n <my-namespace> get pvc <my-pvc> -ojson | jq -r '.spec.storageClassName'`       
+NAME                 PROVISIONER            RECLAIMPOLICY   VOLUMEBINDINGMODE   ALLOWVOLUMEEXPANSION   AGE
+standard (default)   kubernetes.io/gce-pd   Delete          Immediate           true                   28d
+```
+
+In this case `ALLOWVOLUMEEXPANSION` is true, so we can make use of the feature.
+
+To resize the volume run:
+
+```shell
+$ kubectl -n <my-namespace> edit pvc <my-pvc>
+```
+
+And edit `.spec.resources.requests.storage` to the new desired storage size.
+Eventually the PVC status will say "Waiting for user to (re-)start a pod to
+finish file system resize of volume on node."
+
+You can check this with:
```

Comment:
I would add also the command to see all namespaces - make it easier on beginners

Reply:
It's the same runbook as for KubePersistentVolumeFillingUp - https://github.com/prometheus-operator/runbooks/blob/main/content/runbooks/kubernetes/KubePersistentVolumeFillingUp.md

Let's keep it the same, as it would be easier to update in the future.
Your comment might still be helpful, it could be something cool that you submit a PR to the prometheus-operator repo with that change.